src/main/java/uk/gov/cshr/controller/signup/SignupController.java

```diff
@@ -93,14 +99,24 @@ public String sendInvite(Model model,
         }
 
         final String email = form.getEmail();
-
-        if (inviteRepository.existsByForEmailAndStatus(email, InviteStatus.PENDING)) {
+        Optional<Invite> invite = inviteService.findByForEmailAndStatus(email, InviteStatus.PENDING);
```

Comment:
variable invite should be renamed to pendingInvite

Reply:
Done